From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Arjan van de Ven <arjan@linux.intel.com>
Date: Sun, 4 Jun 2017 17:35:40 +0000
Subject: [PATCH] support for .avx2 and .avx512 so files

Signed-off-by: Patrick McCarty <patrick.mccarty@intel.com>
---
 src/unix/dynload.c | 13 +++++++++++++
 1 file changed, 13 insertions(+)

diff --git a/src/unix/dynload.c b/src/unix/dynload.c
index f03d7cb..489edf8 100644
--- a/src/unix/dynload.c
+++ b/src/unix/dynload.c
@@ -77,8 +77,21 @@ static void *loadLibrary(const char *path, int asLocal, int now,
 {
     void *handle;
     int openFlag = 0;
+    char temppath[8192];
 
     openFlag = computeDLOpenFlag(asLocal, now);
+
+    sprintf(temppath, "%s.avx512", path);
+    if (access(temppath, R_OK) == 0 && __builtin_cpu_supports("avx512cd")) {
+	handle = (void *)dlopen(temppath, openFlag);
+	return handle;
+    }
+
+    sprintf(temppath, "%s.avx2", path);
+    if (access(temppath, R_OK) == 0 && __builtin_cpu_supports("avx2")) {
+	handle = (void *)dlopen(temppath, openFlag);
+	return handle;
+    }
     handle = (void *) dlopen(path,openFlag);
 
     return(handle);
