--- R-4.2.1/src/library/base/R/serialize.R~	2020-03-26 23:02:08.000000000 +0000
+++ R-4.2.1/src/library/base/R/serialize.R	2022-08-24 13:59:39.629551972 +0000
@@ -18,7 +18,7 @@
 
 saveRDS <-
     function(object, file = "", ascii = FALSE, version = NULL,
-             compress = TRUE, refhook = NULL)
+             compress = FALSE, refhook = NULL)
 {
     if(is.character(file)) {
 	if(file == "") stop("'file' must be non-empty string")
