diff --git a/config.site b/config.site
index 29daa2c..33f58a2 100644
--- a/config.site
+++ b/config.site
@@ -381,13 +381,13 @@
 
 ## Commands which might need to be set for LTO builds
 ## Typical values for LTO are gcc-ar, gcc-nm and gcc-ranlib
-## AR=
-## NM=
-## RANLIB=
+AR=gcc-ar
+NM=gcc-nm
+RANLIB=gcc-ranlib
 ##
 ## The LTO macro, defaulting to -flto
 ## Could be e.g. -flto=8 for GCC or -flto=thin for clang.
-## LTO= 
+LTO= -flto=8
 ## Ditto for the Fortran compiler, defaulting to the value of LTO.
 ## LTO_FC= 
 ##
